### DISCUSSION TITLE
How do i integrate a custom tensorflow model as a plugin in ant media server ? (Migrated from Issues)

### DISCUSSION THREAD

**Discussion by mekya** [ant-media/Ant-Media-Server#5973] [Q&A]
@[rm75](https://github.com/RMJ5) asked this questions on issues and I've moved here to give faster response
https://github.com/ant-media/Ant-Media-Server/issues/5940

>I want to integrate my custom Tensorflow model into the ant media server as a plugin. Is there any tutorial or documentation for the >same ? I checked the ant media docs but did not find anything comprehensive.

**Comment by lastpeony** [comment]
There is a sample tensorflow plugin which is using a pb model file to do object detection.
https://github.com/ant-media/Plugins/blob/3faa4f659cf93f376c32e9d08153a059a1fc5d45/TensorflowPlugin/src/main/java/io/antmedia/plugin/tensorflow/detection/TFObjectDetector.java#L57

https://github.com/ant-media/Plugins/tree/master/TensorflowPlugin

You can add your own pb file and use similar to the object detector tensorflow plugin
